FastAutoAugment/data.py

```diff
@@ -254,13 +255,17 @@ class Augmentation(object):
     def __init__(self, policies):
         self.policies = policies
 
-    def __call__(self, img):
+    def __call__(self, img, hardness_score=None):
         for _ in range(1):
             policy = random.choice(self.policies)
             for name, pr, level in policy:
                 if random.random() > pr:
                     continue
-                img = apply_augment(img, name, level)
+                if C.get()['hardness']['use']:
+                    assert hardness_score is not None
+                    img = apply_augment(img, name, level, hardness_score)
+                else:
+                    img = apply_augment(img, name, level)
```

Comment:
no need to do this. just do:

```
img = apply_augment(img, name, level, hardness_score)
```

Reply:
We have 2 different booleans. One is C.get()['hardness']['compute'] -- (whether to compute hardness or not). Other is C.get()['hardness']['use'] -- (whehter to use the computed hardness or not). So I had to put this C.get()['hardness']['use'] condition somewhere.